### ISSUE TITLE
Links to Valkey documentation

### ISSUE THREAD

**Issue by aiven-sal** [aiven-sal/valkey-py#4]
Many links in the doc and the source code still point to Redis doc. We can't just point to Vakey's doc in most cases, because Valkey is missing a lot of content.
We can't point to Redis doc forever, so we need to understand how Valkey intends to handle this, maybe we can help.

**Comment by zuiderkwast** [comment]
You can point to files in the valkey-doc repo on github if you want. Example URL: https://github.com/valkey-io/valkey-doc/blob/main/topics/cluster-tutorial.md

It might be better to wait for it to be up on the Valkey website though.

Another option is to refer to valkey man pages names. These will most likely be distributed in debian, fedora, etc. You can try 'make install' of the valkey-doc (see the README).

**Comment by aiven-sal** [comment]
I think we can wait until the website is updated and change the links then.

**Comment by celestehorgan** [comment]
@aiven-sal - Valkey is migrating its website to a new static site generator. As far as I understand, the content in `valkey-io/valkey-docs` will probably move to `valkey-io/valkey-io.github.io`. 

This is a fairly big move so yes, I'd hold off until that's done.

**Comment by zuiderkwast** [comment]
@celestehorgan No, the website is not the owner of the docs. As much as you may want to put the docs in the website repo, developers want it to be in the code repo (because features are developed and documented by the same developers at the same time) and distro packagers want the docs for building man pages.
